README.md

```diff
@@ -29,7 +29,7 @@ $ pivot -r $REGISTRY/os:latest
 OR
 
 ```
-$ docker run --network host -d -w /srv/tree/repo $REGISTRY/os:latest
+$ docker run --network host -d -w /srv/repo $REGISTRY/os:latest
 $ ostree remote add --no-gpg-verify oscontainer http://localhost:8080/repo
```

Comment:
Hi, this line seems to have some inconsistency from the change in index.html?
`ostree remote add --no-gpg-verify os-svc http://os-content.namespace.svc:8080/`

The following(change in `index.html`) works for me while the other doesn't:
```
[root@localhost ~]# ostree remote add --no-gpg-verify os-test http://localhost:8080 
[root@localhost ~]# rpm-ostree rebase -r os-test:openshift/3.10/x86_64/os
1774 metadata, 12350 content objects fetched; 650605 KiB transferred in 217 seconds 
```

Did I miss something? Thanks.

**P.S**: I came across it while testing for https://github.com/openshift/os/issues/157.

Reply:
Yes, you're correct! This should just be `http://localhost:8080` now since the root dir for nginx is already in the repo.